transports/azure-storage-queues/multi-storageaccount-support_registered_endpoint_asq_[8,].partial.md

```diff
@@ -0,0 +1,15 @@
+## Using registered endpoints
```

Comment:
The filename is off.  Shouldn't the range be `[8,)`? I.e. from version 8 (included) and up.